frontend/src/components/common/Grid/Grid.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+
+import GridItem from './GridItem';
+
+const formatAreas = (areas: string[]) => areas.map(area => `'${area}'`).join(' ');
+
+const frGetter = (value: string | number) =>
+  typeof value === 'number' ? `repeat(${value}, 1fr)` : value;
```

Comment:
nit: frGetter sounds kind of obscure since this is generating a string. Some docs or what does fr mean wold help readability

Reply:
yeah - as I mentioned to Devin this PR is low on comments due to time constraints to get it out the door. Can add more comments later

Reply:
Also "fr" is standard CSS grid terminology:

https://css-tricks.com/introduction-fr-css-unit/
https://www.digitalocean.com/community/tutorials/css-css-grid-layout-fr-unit

Reply:
But you are right, the intent of this helper function should be documented

Reply:
Why not just use the package instead of copying/pasting code?
https://www.npmjs.com/package/@react-css/grid?activeTab=readme

Reply:
We could but would be one more dep. Happy to switch (when we need it)

Reply:
Oh I remembered why now. That one is **not** a styled component, so re-styling it with styled might not be straightforward

Reply:
This one is a styled component (but I liked the API from the other package better) - https://github.com/azz/styled-css-grid